Source/Scene/Cesium3DTile.js

```diff
@@ -255,25 +283,28 @@ define([
      * DOC_TBA
      */
     Cesium3DTile.prototype.visibility = function(cullingVolume) {
-        return cullingVolume.computeVisibilityWithPlaneMask(this._orientedBoundingBox, this.parentPlaneMask);
+        var boundingVolume = defaultValue(this._orientedBoundingBox, this._tileBoundingSphere);
```

Comment:
Instead of doing calls like this every frame:
```javascript
defaultValue(this._orientedBoundingBox, this._tileBoundingSphere)
```
Couldn't we just assign to a new `this._boundingVolume` in the constructor.  We might not even need to keep the "less preferred" bounding volumes in memory.